client/scss/components/_button.scss

```diff
@@ -321,35 +321,42 @@
 }
 
 .w-header-button {
```

Comment:
Once we implement the "switch icon on hover" effect, this technically should share the exact same styles as the chooser's e.g. "Add an image" button.